swap_meet/item.py

```diff
@@ -0,0 +1,39 @@
+class Item:
+
+    def __init__(self, category = "", condition = 0, age = 0):
+        if category is None:
+            self.category = ""
+        else:
+            self.category = category
+        if condition is None:
+            self.condition = 0
+        else:
+            self.condition = condition
+        if age is None:
+                self.age = 0
+        else:
+            self.age = age
```

Comment:
this works good! But our default parameters, like `category=""` already do what the if-else statements do. so let's let the default parameters do all the work.

```suggestion
    def __init__(self, category = "", condition = 0, age = 0):
        self.category = category
        self.condition = condition
        self.age = age
```

now category will be assigned `""` if the user doesn't add in an argument. It will do what if-else statements are doing for us!